experimental/add_to_app/android_using_plugin/app/src/main/java/dev/flutter/example/androidusingplugin/MainActivity.kt

```diff
@@ -5,23 +5,16 @@
 package dev.flutter.example.androidusingplugin
 
 import android.os.Bundle
-import android.widget.Button
-import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
 import io.flutter.embedding.android.FlutterActivity
+import kotlinx.android.synthetic.main.activity_main.*
```

Comment:
I actually avoided using the extensions on purpose, since I've read they're no longer the recommended practice. 

https://old.reddit.com/r/androiddev/comments/ala9p2/why_kotlinx_synthetic_is_no_longer_a_recommended/efdvpkg/
https://android-review.googlesource.com/c/platform/frameworks/support/+/882241

Reply:
Ah ok, good to know. Thanks for the info.